ui/main/src/app/business/view/activityarea/activityarea.view.ts

```diff
@@ -39,9 +44,33 @@ export class ActivityAreaView {
                         activityAreaLine.entityId = entity.id;
                         activityAreaLine.entityName = entity.name;
                         activityAreaLine.isUserConnected = entitiesConnected?.includes(entity.id);
-                        this.activityAreaPage.lines.push(activityAreaLine);
+
+                        if (entity?.parents) {
+                            entity.parents.forEach((parentId) => {
+                                const parentEntity = EntitiesService.getEntity(parentId);
+                                if (parentEntity?.roles && parentEntity.roles.includes(RolesEnum.ACTIVITY_AREA_GROUP)) {
+                                    this.isEntityAlreadyACluster(parentEntity.name)
+                                        ? this.addLineToCluster(
+                                              this.activityAreaClusters.get(parentEntity.name),
+                                              activityAreaLine
+                                          )
+                                        : this.activityAreaClusters.set(
+                                              parentEntity.name,
+                                              new ActivityAreaEntityCluster(parentEntity.name, [activityAreaLine])
+                                          );
+                                } else {
+                                    this.addLineToCluster(this.activityAreaOrphanEntitiesCluster, activityAreaLine);
+                                }
```

Comment:
the add to orphan enitities cluster shall be done as well when entity has no parents and not only when parents has not the role ACTIVITY_AREA_GROUP

Add two units tests to check this behavior